### PR TITLE
Preparing release 3.1.0

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,10 +4,12 @@ Changelog
 This document describes changes between each past release.
 
 
-3.0.3 (unreleased)
+3.1.0 (2020-09-29)
 ==================
 
-- Nothing changed yet.
+- Add support for language configuration
+- Remove the jQuery dependency and fix multiples errors around that
```
Comment (claudep) [inline CHANGELOG.rst:11]: I would add that the `TINYMCE_INCLUDE_JQUERY` setting is no more.

**Review by claudep — COMMENTED** [review]
Thanks for preparing the release!
Should we upgrade to the latest TinyMCE code before the release?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,10 @@ This document describes changes between each past release.
 
 - Add support for language configuration
 - Remove the jQuery dependency and fix multiples errors around that
+
+  .. note::
+     As a consequence, ``TINYMCE_INCLUDE_JQUERY`` settings has been removed.
```
Comment (claudep) [inline CHANGELOG.rst:14]: +1, but you can remove the `s` to settings :-)

**Comment by Natim** [comment]
>  Should we upgrade to the latest TinyMCE code before the release?

@claudep if you feel like opening a PR I can wait yes !

**Review by claudep — APPROVED** [review]

**Pull request merged**